Horde3D/Source/ColladaConverter/main.cpp

```diff
@@ -272,6 +277,14 @@ int main( int argc, char **argv )
 			log( "Parsing dae asset '" + assetList[i] + "'..." );
 			if( !daeDoc->parseFile( sourcePath ) )
 				return 1;
+
+			if( useMaterialId )
```

Comment:
Please add a comment what this code is for - a single sentence should be good. For other coders, who have not encountered this problem, it may lead to questions, why internal collada states are modified.